src/Responder/LtiInstance/LtiInstanceModel.php

```diff
@@ -62,7 +62,7 @@ public function jsonSerialize(): array
 
     public function fillFromEntity(LtiInstance $entity): self
     {
-        $this->id = $entity->getId();
+        $this->id = (int)$entity->getId();
```

Comment:
It looks strange that the method getId() can return something other than integer value. Maybe better consider redesign code in the LtiInstance class?

Reply:
It's doctrine model. By docs it could return `null` value but it's not real case. So i just fed analyser with typecast.